.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/**
+
+!node_modules/cylc-action-utils.js
+# Have put that file in node_modules so that we don't have to specify the path to it every time we require() it
```

Comment:
When I installed the project it deleted this file.

```console
$ yarn install >/dev/null
$ git status
HEAD detached at ronnie/content
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	deleted:    node_modules/cylc-action-utils.js

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	yarn.lock
```

Reply:
It doesn't happen when using npm. Should I just put a note in README saying "use npm instead of yarn"?

Reply:
So long as it works I'm happy.